src/openms/source/FORMAT/HANDLERS/MzIdentMLHandler.cpp

```diff
@@ -971,7 +971,7 @@ namespace OpenMS::Internal
       //--------------------------------------------------------------------------------------------
       os << "<cvList>\n"
          << "\t<cv id=\"PSI-MS\" fullName=\"Proteomics Standards Initiative Mass Spectrometry Vocabularies\" "
-         << "uri=\"https://raw.githubusercontent.com/HUPO-PSI/psi-ms-CV/master/psi-ms.obo\" "
+         << "uri=\"http://purl.obolibrary.org/obo/ms/psi-ms.obo\" "
```

Comment:
There is really no secure connection to that obo? That's really sad..

Reply:
Yeah annoying but I made it now work with http as well.